### PR TITLE
test: Use wait_text to check an empty string

### PR DESCRIPTION
**Pull request by skobyda**
Using wait_in_text(selector, "") always validates as true (since empty string is always subset of any string), rendering the test useless.

### CODE REVIEW

**Comment by martinpitt** [comment]
> wait_js_cond(ph_text_is("#pool-myPoolTwo-system-volume-VolumeOne-usedby","")): Uncaught (in promise) Error: actual text: subVmTest1

... and it fails

**Comment by skobyda** [comment]
> > wait_js_cond(ph_text_is("#pool-myPoolTwo-system-volume-VolumeOne-usedby","")): Uncaught (in promise) Error: actual text: subVmTest1
> 
> ... and it fails

So the test was really useless

File: test/machineslib.py [hunk 1]
```diff
@@ -235,6 +235,19 @@ def run_admin(self, cmd, connectionName='system'):
         else:
             return m.execute(cmd)
 
+    def deleteDisk(self, target, vm_name="subVmTest1"):
```
Comment (martinpitt) [inline test/machineslib.py:238]: You just defined this function without using this anywhere. Forgot to commit something?

**Review by martinpitt — CHANGES_REQUESTED** [review]
Thanks for fixing the test properly!

**Review by martinpitt — APPROVED** [review]
Thanks!

**Pull request merged**